framework/src/main/java/org/checkerframework/common/value/ValueTreeAnnotator.java

```diff
@@ -257,6 +258,21 @@ public Void visitTypeCast(TypeCastTree tree, AnnotatedTypeMirror atm) {
       if (oldAnno == null) {
         return null;
       }
+
+      boolean isUnsigned = false;
+      // I would like to call ((AnnotatedTypeTree) castTree).hasAnnotation(Unsigned.class), but
+      // `Unsigned` is in the checker package and this code is in the common package.
+      List<? extends AnnotationTree> annoTrees =
+          TreeUtils.getExplicitAnnotationTrees(null, tree.getType());
+      List<AnnotationMirror> annos = TreeUtils.annotationsFromTypeAnnotationTrees(annoTrees);
+      for (AnnotationMirror am : annos) {
```

Comment:
This for loop can be replaced with:
be `isUnsigned = AnnotationUtils#containsSameByName(annos, "org.checkerframework.checker.signedness.qual.Unsigned")`.